bin/run_docker.py

```diff
@@ -178,42 +241,29 @@ def run_docker(
 
         log_text = container
 
+    # Capture any errors that may occur during the attempt to run the container
     except Exception as e:
         log_text = str(e).replace("\\n", "\n")
         create_log_file(
-            log_file_name=log_file_name, log_file_path=log_file_path, log_text=log_text
+            log_file_name=log_file_name, log_file_path=output_path, log_text=log_text
         )
 
         raise
```

Comment:
Shouldn't this `raise` be removed? We don't want this process to fail, even if the docker execution fails because the submitter will receive no feedback if their submission does not make it all the way through the workflow.

Reply:
good catch! Updated